definitions/private-law/json/CaseEventToFields/DraftOrderAdmin/Page14-Hearing Data-solicitor/CaseEventToFields.json

```diff
@@ -9,7 +9,7 @@
     "PageID": 14,
     "PageDisplayOrder": 14,
     "ShowSummaryChangeOption": "Y",
-    "PageShowCondition": "doYouWantToEditTheOrder=\"Yes\" AND isOrderCreatedBySolicitor=\"Yes\" AND isHearingPageNeeded=\"Yes\""
+    "PageShowCondition": "doYouWantToEditTheOrder=\"DO_NOT_SHOW\" AND isOrderCreatedBySolicitor=\"Yes\" AND isHearingPageNeeded=\"Yes\""
```

Comment:
Since this page is not used anymore. Remove rest of the fields from this page, as we can reduce the size of the ccd config file.

And also update the folder name to say that its not in use

Reply:
Done